packages/block-library/src/navigation-menu-item/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+
+const { name } = metadata;
+export { metadata, name };
+
+export const settings = {
+	title: __( 'Menu Item (Experimental)' ),
+
+	parent: [ 'core/navigation-menu' ],
+
+	icon: 'admin-links',
+
+	description: __( 'Add a page, link, or other item to your Navigation Menu.' ),
+
+	supports: {},
+
+	edit,
+
+	save( { attributes } ) {
```

Comment:
The direction is to have all client-side specific cod in its own file. `save` should be moved to `save.js` file. This will allow to get rid of `index.js` completely at some point.